<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMS Robot GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="streamButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start stream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectWiFiButton">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>320</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>229</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>235</green>
          <blue>231</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Camera module</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="messageEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>411</width>
      <height>192</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>296</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>235</green>
          <blue>231</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Motor and servo control</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="RobotImageRenderer" name="robotRenderer" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="motorSliderEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>20%</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Motor speed:</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="motorConnectionComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>WiFi</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="replayButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="WiFiIcon" name="wifiIcon" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="motorCompEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMotorCompButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendSteeringCompButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="steeringCompLinEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Steering comp.</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Motor comp.</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="steeringCompOffsEdit_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="errorButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Error report</string>
    </property>
   </widget>
   <widget class="WiFiIcon" name="wifiIconRPI" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>235</green>
          <blue>231</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MB WiFi</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>235</green>
          <blue>231</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RPi WiFi</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="photoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Snap photo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectWiFiButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoscroll_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto scroll</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Stream_settings"/>
    <addaction name="actionMotor_Board_Settings"/>
    <addaction name="actionPlot"/>
    <addaction name="action_Logging_options"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Manual"/>
    <addaction name="action_Doxygen"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Stream_settings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Camera Module Settings</string>
   </property>
  </action>
  <action name="actionClear_logs">
   <property name="text">
    <string>Clear logs</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionMotor_Board_Settings">
   <property name="text">
    <string>Motor Board Settings</string>
   </property>
  </action>
  <action name="action_Manual">
   <property name="text">
    <string>&amp;Manual</string>
   </property>
  </action>
  <action name="action_Doxygen">
   <property name="text">
    <string>&amp;Doxygen</string>
   </property>
  </action>
  <action name="action_Logging_options">
   <property name="text">
    <string>&amp;Logging options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RobotImageRenderer</class>
   <extends>QWidget</extends>
   <header>robotimagerenderer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WiFiIcon</class>
   <extends>QWidget</extends>
   <header>wifiicon.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
